src/javascript/binary/static/viewbalance/viewbalance.ui.js

```diff
@@ -0,0 +1,58 @@
+
+var ViewBalanceUI = (function(){
+    var colsName = ["loginid", "currency", "balance"];
+
+    function createEmptyBalanceTable(){
+        var header = [
+            Content.localize().textLoginID,
+            Content.localize().textCurrency,
+            Content.localize().textBalance
+        ];
+        var metadata = {
+            cols: colsName,
+            id: "bal-table"
+        };
+        var data = [];
+        var $table = Table.createFlexTable(data, metadata, header);
+
+        return $table;
+    }
+
+    function createBalancePopup(){
+        var $popupDiv = $("<div></div>", {class: "popup-div", id: "balance-container"});
+        var $balTitle = $("<h1></h1>", {text: Content.localize().textBalances});
+        var $br = $("<br>");
+        var $table = createEmptyBalanceTable();
+
+        var $span = $("<span></span>", {class: "button"});
+        var $button = $("<button></button>", {
+            class: "button",
+            id: "close-balances",
+            text:Content.localize().textContinueTrading
+        });
+        $span.append($button);
+
+        $popupDiv.append($balTitle).append($br).append($table).append($span);
```

Comment:
I am totally fine with this solution. 
It is fast enough + maintainable. A good balance.